tests/test_main.py

```diff
@@ -15,6 +17,18 @@ def test_predict() -> None:
         assert len(output[0]) > 0, "Expected non-empty output"
 
 
+def test_evaluation() -> None:
+    # Test Case: Valid Scores in the range of 0 and 1
+    data = Path(__file__).parent.joinpath("../data/data.jsonl").resolve()
+    with jsonlines.open(data) as reader:
+        items = [item for item in reader]
+        labels = [item["output"] for item in items]
+
+    bleu, meteor = evaluate_documentation(labels, labels)
+    assert bleu >= 0 and bleu <= 1, "BLEU score should be between 0 and 1"
```

Comment:
Are bleu, meteor==1 when label==prediction?

It would actually be a bit clearer if you hard code some examples and assert specific values. Eg. all tokens match, extra token in the prediction, missing token in the prediction, etc.

Reply:
Added more tests cases, and code works as expected.